css/css-grid/layout-algorithm/grid-flex-track-intrinsic-sizes-003.html

```diff
@@ -30,57 +32,160 @@
+
+// Item spanning a fixed flexible track
+checkTrackSizes(1, "minmax(0, 0fr)", "0px");
+checkTrackSizes(1, "minmax(0, .5fr)", "30px");
+checkTrackSizes(1, "minmax(0, 1fr)", "60px");
+checkTrackSizes(1, "minmax(0, 2fr)", "60px");
+checkTrackSizes(1, "minmax(75px, 1fr)", "75px");
+
+// Item spanning 2 auto flexible tracks
+checkTrackSizes(2, "0fr 0fr", "0px 0px");
+checkTrackSizes(2, "0fr 1fr", "0px 60px");
+checkTrackSizes(2, "1fr 0fr", "60px 0px");
+checkTrackSizes(2, "1fr 1fr", "30px 30px");
+checkTrackSizes(2, "1fr 3fr", "15px 45px");
+checkTrackSizes(2, "0fr 0fr 1fr", "0px 0px 60px");
+
+// Item spanning 2 fixed flexible tracks
+checkTrackSizes(2, "minmax(0, 0fr) minmax(0, 0fr)", "0px 0px");
+checkTrackSizes(2, "minmax(0, 0fr) minmax(0, 1fr)", "0px 60px");
+checkTrackSizes(2, "minmax(15px, 0fr) minmax(0, 1fr)", "15px 45px");
+checkTrackSizes(2, "minmax(20px, 1fr) minmax(0, 1fr)", "30px 30px");
+checkTrackSizes(2, "minmax(40px, 1fr) minmax(0, 1fr)", "40px 20px");
+
+// Item spanning an auto flexible track and a fixed flexible track
+checkTrackSizes(2, "0fr minmax(0, 0fr)", "0px 0px");
+checkTrackSizes(2, "0fr minmax(0, 1fr)", "0px 60px");
+checkTrackSizes(2, "1fr minmax(0, 1fr)", "30px 30px");
+checkTrackSizes(2, "1fr minmax(25px, 1fr)", "30px 30px");
+
+// Item spanning an auto flexible track and an intrinsic non-flexible track
+checkTrackSizes(2, "0fr auto", "0px 0px");
+checkTrackSizes(2, "1fr auto", "60px 0px");
+checkTrackSizes(2, "1fr max-content", "60px 0px");
+checkTrackSizes(2, "1fr min-content", "60px 0px");
+
+// Item spanning a fixed flexible track and an intrinsic non-flexible track
+checkTrackSizes(2, "minmax(0, 0fr) auto", "0px 0px");
+checkTrackSizes(2, "minmax(0, 1fr) auto", "60px 0px");
+checkTrackSizes(2, "minmax(25px, 0fr) auto", "25px 0px");
+checkTrackSizes(2, "minmax(25px, 1fr) auto", "60px 0px");
+
+// Now check for distributing min content //////////////////////////////////////
+item.style.minWidth = "min-content"; // min-content = 100px > grid
+testset = 'min-content min item'
+
+// Item spanning an auto flexible track
 checkTrackSizes(1, "0fr", "100px");
 checkTrackSizes(1, "1fr", "100px");
 checkTrackSizes(1, "2fr", "100px");
 
 // Item spanning a fixed flexible track
 checkTrackSizes(1, "minmax(0, 0fr)", "0px");
-checkTrackSizes(1, "minmax(0, .5fr)", "25px");
-checkTrackSizes(1, "minmax(0, 1fr)", "50px");
-checkTrackSizes(1, "minmax(0, 2fr)", "50px");
+checkTrackSizes(1, "minmax(0, .5fr)", "30px");
+checkTrackSizes(1, "minmax(0, 1fr)", "60px");
+checkTrackSizes(1, "minmax(0, 2fr)", "60px");
 checkTrackSizes(1, "minmax(75px, 1fr)", "75px");
 
-// Item spanning 2 intrinsic flexible tracks
+// Item spanning 2 auto flexible tracks
 checkTrackSizes(2, "0fr 0fr", "50px 50px");
 checkTrackSizes(2, "0fr 1fr", "0px 100px");
 checkTrackSizes(2, "1fr 0fr", "100px 0px");
 checkTrackSizes(2, "1fr 1fr", "50px 50px");
 checkTrackSizes(2, "1fr 3fr", "25px 75px");
-checkTrackSizes(2, "0fr 0fr 1fr", "50px 50px 0px");
```

Comment:
This would be `50px 50px 0px` for columns and `0px 0px 60px` for rows.

Reply:
Why would it be "50px 50px 0px" for columns?

Reply:
Because the item spans the 1st and 2nd columns, and has `min-width: 100px`, so that's its minimum contribution, which gets distributed into the base sizes of the 1st and 2nd columns per https://drafts.csswg.org/css-grid/#track-size-intrinsic-min. Then there is no free space, so the last column stays as 0.

Reply:
Maybe you were thinking about `checkTrackSizes(3, "0fr 0fr 1fr", "0px 0px 100px");`?

Reply:
Hah. Yes, you're exactly right, that was I was thinking. :) Will fix.